src/index/FTSAlgorithms.cpp

```diff
@@ -113,6 +113,84 @@ Index::WordEntityPostings FTSAlgorithms::intersect(
   return resultWep;
 }
 
+// _____________________________________________________________________________
+Index::WordEntityPostings FTSAlgorithms::crossIntersect(
+    const Index::WordEntityPostings& matchingContextsWep,
+    const Index::WordEntityPostings& eBlockWep) {
+  // Example:
+  // matchingContextsWep.wids_: 3 4 3 4 3
+  // matchingContextsWep.cids_: 1 4 5 5 7
+  // -----------------------------------
+  // eBlockWep.cids_          : 4 5 5 8
+  // eBlockWep.eids_          : 2 1 2 1
+  // ===================================
+  // resultWep.cids_          : 4 5 5 5 5
+  // resultWep.wids_          : 4 3 4 3 4
+  // resultWep.eids_          : 2 1 2 2 1
```

Comment:
This example really helps. The `matchingConstexWep` should also start with the `cids_` before the `wids_` for
readability. So this intersects the texts that match because of the `words` with the texts that match because of the `entities`. Interesting.

Reply:
(The example should be part of the documentation in the header).